ts/DataGrid/Accessibility/Accessibility.ts

```diff
@@ -202,7 +210,13 @@ class Accessibility {
      * The order of the sorting.
      */
     public userSortedColumn(order: ColumnSortingOrder): void {
-        const messages = this.dataGrid.options?.accessibility?.sorting;
+        const messages = this.dataGrid.options?.lang
+            ?.accessibility?.sorting?.announcements;
```

Comment:
Why do we use the messages variable, when we refer to announcements.

Its also strange formatting (or its a GitHub issue).

Reply:
The name of the first argument of the `announce` method is `msg` - message.

Reply:
Hm...but it refers to variable.